app/controllers/users_controller.rb

```diff
@@ -0,0 +1,61 @@
+class UsersController < ApplicationController
+  before_action :set_user, only: %i[ show edit update destroy ]
+
+  def index
+    @users = User.all
+  end
+
+  def show
```

Comment:
Create empty actions in application controller and overwrite in other controllers only if you need them

Reply:
Shouldn't I just remove them?